rviz_common/src/rviz_common/temp/default_plugins/displays/grid_display.cpp

```diff
@@ -54,7 +54,7 @@
 #include "../../../selection/selection_manager.hpp"
 
 using rviz_rendering::Grid;
-using namespace rviz_common::properties;
+using namespace rviz_common::properties;  // NOLINT revisit later on
```

Comment:
Just replace with explicit using statements, e.g. `using rviz_common::properties::x; using rviz_common::properties::y;` etc...

This can be done in a follow up pr, but it should be done.

Reply:
Follow up PR is in preparation.